python/veles/objects/chunk.py

```diff
@@ -38,3 +51,45 @@ def __getitem__(self, arg):
                     else self._end - self._start)
             stop += self._start
             return self._blob[slice(start, stop, arg.step)]
+
+    def create_chunk_item(self, name, num_elements=1, start=None,
+                          update_cursor=True,
+                          endianness=RepackEndian.LITTLE,
+                          bit_width=8, high_pad=0, low_pad=0,
+                          mode=FieldHighMode.FIXED,
+                          shift=0,
+                          sign_mode=FieldSignMode.UNSIGNED,
+                          float_mode=FieldFloatMode.IEEE754_SINGLE,
+                          float_complex=False,
+                          string_mode=FieldStringMode.STRING_RAW,
+                          string_encoding=FieldStringEncoding.ENC_RAW,
+                          type_name=""):
+        # TODO big docstring
+        req = network_pb2.Request()
+        req.type = network_pb2.Request.ADD_CHUNK_ITEM
+        req.id.extend(self._id_path)
+        # TODO other types than just field
+        req.chunk_item.type = ChunkDataItemTypes.FIELD
+        if start is None:
+            start = self._cursor
+        req.chunk_item.start = start
+        req.chunk_item.name = name
+        req.chunk_item.repack.endian = endianness
+        req.chunk_item.repack.width = bit_width
+        req.chunk_item.repack.high_pad = high_pad
+        req.chunk_item.repack.low_pad = low_pad
+        req.chunk_item.num_elements = num_elements
+        req.chunk_item.high_type.mode = mode
+        req.chunk_item.high_type.shift = shift
+        req.chunk_item.high_type.sign_mode = sign_mode
+        req.chunk_item.high_type.float_mode = float_mode
+        req.chunk_item.high_type.float_complex = float_complex
+        req.chunk_item.high_type.string_mode = string_mode
+        req.chunk_item.high_type.string_encoding = string_encoding
+        req.chunk_item.high_type.type_name = type_name
+        results = self._client._send_req(req)
+
+        self.items_from_another(results[0])
+        if update_cursor:
+            self._cursor = self.items[-1].end
```

Comment:
What's the ordering of ChunkDataItems? Are they sorted by creation order? If not this may be assuming too much.